### PR TITLE
[REFACTOR] Move the Hacktoberfest example to the Demo folder

### PR DESCRIPTION
**Pull request by csouchet**
**Live environment of examples**

https://cdn.statically.io/gh/process-analytics/bpmn-visualization-examples/move_hacktoberfest_example_in_demo_folder/examples/index.html

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,7 +85,6 @@ Custom BPMN Theme features will be progressively added to `bpmn-visualization`.
 - [Custom user task icon](examples/custom-bpmn-theme/custom-user-task-icon/README.md) - use your own icon
 - [Custom colors](examples/custom-bpmn-theme/custom-colors/README.md) - custom defaults, specific to BPMN element types
 - [Custom fonts](examples/custom-bpmn-theme/custom-fonts/README.md) - custom defaults, specific to BPMN element types
-- [Hacktoberfest themes](examples/custom-bpmn-theme/hacktoberfest-diagram/README.md) - special Hacktoberfest diagram with Hacktoberfest colors
```
Comment (tbouffard) [inline README.md:88]: we didn't do it for the monitoring demo, but we could add a dedicated section for all demos as we now have readme for some of them.
In the repo readme, we could add just a link to a new readme in the demo folder that will ref the demo

File: demo/hacktoberfest-custom-themes/README.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
-# Hacktoberfest diagram
+# Hacktoberfest custom themes
 
 **DISCLAIMER: this extension point is very experimental and is subject to change**.
```
Comment (tbouffard) [inline demo/hacktoberfest-custom-themes/README.md:3]:
```suggestion
**DISCLAIMER: this demo uses extension points that are very experimental and are subject to change**.  
```

**Review by tbouffard — APPROVED** [review]
✅ tested on the surge enviroment
ℹ️ minor things could be improved

File: README.md [hunk 3]
```diff
@@ -31,6 +31,13 @@ You will find both
 [bpmn-visualization](https://github.com/process-analytics/bpmn-visualization-js) repository.
 - the examples described in more details below in this page. They demonstrate how to use a single feature.
 
+## Demo
```
Comment (csouchet) [inline README.md:34]: @tbouffard Is it ok four you ?

File: README.md (hunk 1 above)
Comment (csouchet) [reply README.md:88]: Done

File: README.md (hunk 3 above)
Comment (tbouffard) [reply README.md:34]: 👍🏿 perfect

Comment (tbouffard) [reply README.md:34]: the title miss an emoji for consistency with others.

Comment (tbouffard) [reply README.md:34]: the `a `demo` that shows several features of `bpmn-visualization` at the same time. The sources of the demo are available in the` sentence above should be reworked.

Several demo + remove ref to the repository

Comment (csouchet) [reply README.md]: Yes, I forgot this 🙁

**Pull request merged**